database/layer/admin-client.js

```diff
@@ -107,9 +105,26 @@ const deleteClientByClientId = async (clientId) => {
   }
 }
 
+/**
+ * Retrieve a list of all clients
+ *
+ * @returns {Promise<Array>} - A promise that resolves to an array of clients
+ * @throws {Error} - If there is an error during the retrieval process
+ */
+const getClients = async () => {
+  try {
+    return await usherDb('clients')
+      .select('*')
+      .returning('*')
```

Comment:
@k1ch do you need the returning call here? Doesn't the select return the data? 
I think returning is used for just insert/update statements.

Reply:
You are right Michael. Good catch!